CODE_REVIEWS.md

```diff
@@ -14,4 +14,18 @@
   * Screenshots (for visual changes or new additions)
 2. Reviews provide comments and the author responds / makes changes. Repeat until LGTM.
 3. One or more of the reviewers applies the "LGTM" label.
-4. The party responsible for merging PRs will do so. 
+4. Once the LGTM label is applied, either the author or the reviewer can add the "merge-ready"
+   label to indicate that the PR is ready to be merged.
+5. The party responsible for merging PRs will do so.
+
+## How PRs are merged
+The team has a weekly rotation for the "caretaker" who is responsible for merging PRs. Before being
+merged, the caretaker runs PRs through Google's internal presubmit system. Due to the volume of
```

Comment:
Should we mention something about how the volume of tests adds an extra layer of vetting before releasing to the rest of the community?  Canary, etc.